tests/conftest.py

```diff
@@ -12,3 +12,18 @@ def make_simple_data():
     """Make a simple X, y dataset."""
     X, y = pd.DataFrame(np.ones((100, 2))), pd.Series(np.ones(100))
     return X, y
+
+
+@pytest.fixture
+def linear_causal_data():
```

Comment:
Am I right that this is being by `test_dependence.test_ice`? So does using pytest fixtures ambiently set the value of test functions' parameters *across modules*? That seems quite brittle

Reply:
I'm not sure I quite get what you mean by this comment? Each test should get its own "fixture" (copy of the cached data) - or do you mean something else?

Reply:
Sorry, that was a poor way to explain my confusion. Clearly `pytest` is calling e.g. `test_dependence.test_ice()` using the fixture `linear_causal_data` as an argument. I found it disconcerting that this induces an implicit coupling between `conftest.py` and `test_dependence.py`, when neither imports the other.
I've since read [the relevant documentation](https://docs.pytest.org/en/latest/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files) and realised that `conftest.py` is a local config file and `pytest` manages scope for us, so I can understand what's happening now and see that this is idiomatic